podpac/core/coordinates/coordinates.py

```diff
@@ -856,7 +857,7 @@ def json(self):
 
         return json.dumps(self.definition, separators=(",", ":"), cls=podpac.core.utils.JSONEncoder)
 
-    @property
+    @cached_property

```

Comment:
@jmilloy I ended up caching this because it did cause quite a bit of slowdown -- it unfortunately leads to some hard-to-find caching bugs when running interactively.

Reply:
Yeah... hmmm. I'm sort of surprised about the caching bugs, and very concerned about introducing them. I'm surprised because `cached_property` doesn't use the podpac cache (right?), it just uses a private attribute in the class. Can you demo what kind of bug you were seeing?

What part is slow? Is it the hash_alg, or hex_digest, or what? I wonder if we can locally cache the results of the hash_alg or whatever the sticking point is in order to avoid the slow down and the bugs. E.g....

```python3
hash_alg_cache = {}
...
def cached_hash_alg(s):
    global hash_alg_cache
    if s not in hash_alg_cache:
        hash_alg_cache[s] = hash_alg(s)
    return hash_alg_cache[s]
```

Reply:
I didn't see any actual bugs... but I recall when making ui's it's convenient to overwrite certain traits (in which case the user supposedly knows what they are doing). But in those cases no modification actually changed the output of eval because it was just grabbing data from cache (because the hash stayed the same... in my memory example it was actually the definition that was cached.)... I'm not sure exactly what part was slow, but caching this saved 0.5 a second on a request.